src/components/List/ListItem/index.ts

```diff
@@ -1,4 +1 @@
 export { ListItem, Props as ListItemProps } from './ListItem';
```

Comment:
List story is broken now, ones that are not broken looks just weird, but not sure if it is related to the current PR👀
<img width="521" alt="Screenshot 2024-10-15 at 14 50 33" src="https://github.com/user-attachments/assets/b2f3c43c-a891-48e0-b14d-f6d3a6923b4a">